### PR TITLE
Localize cloud-native apps and tech to fr

### LINKED ISSUE #1260: [FR] Localize cloud-native-tech.md
Localization of cloud-native-tech.md into french.

### PR DESCRIPTION
**Pull request by Krast76**
Fix #1260 and #1258.

@fydrah @huats This PR needs a special focus, because I translated some important words (such as scale etc).
I think we have to review it together at our next meeting.

### CODE REVIEW

File: content/fr/cloud-native-tech.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: Technologies Cloud Native
+status: Completed
+category: Concept
+tags: ["fundamentals", "", ""]
+---
+
+## Ce que c'est
+
+Les technologies Cloud native, aussi référencées comme ensemble des technologies cloud native, sont les technologies
```
Comment (fydrah) [inline content/fr/cloud-native-tech.md:10]: "ensemble des technologies cloud natives" --> "comme la pile des technologies cloud natives"

File: content/fr/cloud-native-apps.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+category: concept
+tags: ["application", "", ""]
+---
+
+## Ce que c'est
+
+Les applications Cloud Native sont des applications conçues pour tirer avantage des innovations du [cloud computing](/cloud-computing/).
+Ces applications s'intègrent facilement avec leurs architectures cloud respective, en tirant avantage des ressources proposées par le Cloud et de leur capacité de [mise à l'échelle](/scalability/).
+Il fait aussi référence aux applications qui tirent avantage d'infrastructure pilotée par le cloud computing.
+Aujourd'hui, les applications Cloud native incluent les applications qui fonctionnent dans le centre de données des fournisseurs de cloud ainsi que les applications qui tournent sur des plateformes hébergées dans des centres de données traditionnelles.
```
Comment (fydrah) [inline content/fr/cloud-native-apps.md:13]: "tournent sur des plateformes hébergées dans des centres de données traditionnelles." --> "tournent sur des plateformes hébergées dans les centres de données internes aux entreprises."

File: content/fr/cloud-native-tech.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+title: Technologies Cloud Native
```
Comment (fydrah) [inline content/fr/cloud-native-tech.md:2]: 'Natives'

File: content/fr/cloud-native-tech.md (hunk 1 above)
Comment (fydrah) [inline content/fr/cloud-native-tech.md:10]: 'Natives'

File: content/fr/cloud-native-tech.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+de domaines différents, il touche.
+Mais à un niveau élevé, ils relèvent un ensemble principal de défis :
+Les inconvénients des modèles d'exploitation informatiques traditionnels.
+Les défis incluent les difficultés à créer des applications évolutives, tolérantes aux pannes et qui sont
+capable de se réparer toute seule. Il adresse également l'utilisation inefficace de ressources entre autres.
+
+## Quel en est l'utilité
+
+Bien que chaque technologie réponde à un problème très spécifique,
+en tant que groupe, les technologies cloud native permettent des systèmes faiblement couplés qui sont résilient, gérables et observable.
```
Comment (fydrah) [inline content/fr/cloud-native-tech.md:30]: 'natives'

**Comment by Krast76** [comment]
Well spotted @huats, fixed.

**Review by fydrah — CHANGES_REQUESTED** [review]

File: content/fr/cloud-native-apps.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+title: Application Cloud Native
+status: Completed
+category: concept
+tags: ["application", "", ""]
+---
+
+## Ce que c'est
+
+Les applications Cloud Native sont des applications conçues pour tirer avantage des innovations du [cloud computing](/cloud-computing/).
+Ces applications s'intègrent facilement avec leurs architectures cloud respective, en tirant avantage des ressources proposées par le Cloud et de leur capacité de [mise à l'échelle](/scalability/).
```
Comment (huats) [inline content/fr/cloud-native-apps.md:11]: respectives

File: content/fr/cloud-native-apps.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+Ce terme fait aussi référence aux applications qui tirent avantage d'infrastructure pilotée par le cloud computing.
+Aujourd'hui, les applications Cloud native incluent les applications qui fonctionnent dans le centre de données des fournisseurs de cloud ainsi que les applications qui tournent sur des plateformes hébergées dans les centres de données internes aux entreprises.
```
Comment (huats) [inline content/fr/cloud-native-apps.md:13]: Native

File: content/fr/cloud-native-apps.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+Chaque centre de données dispose de services qui  [associent étroitement ](/tightly-coupled-architectures/) les applications à des environnements spécifiques, qui se basent souvent sur des infrastructures déployées à la main, par exemple des [machines virtuelles](/virtual-machine/) et des services. Cela contraint les développeurs et leurs applications à un déploiement dans ce centre de données spécifique.
+Les applications qui n'ont pas été conçues pour tirer avantage des environnements Cloud ne pourront pas bénéficier de ses avantages tels que la résilience et la mise à l'échelle.
+Par exemple, les applications nécessitant une intervention manuelle pour démarrer correctement ne pourront pas se mettre à l'échelle automatiquement ou automatiquement redémarrées en cas d'erreur. 
+
+## Quel en est l'utilité
+
+Bien qu'il n'y ait pas de solution miracle pour qu'une application soit définie comme cloud native, les applications
+Cloud Native présentent des points en commun.
+Les applications Cloud Native sont résilientes, elles sont manipulables et tirent profit de l'ensemble des services Cloud dans lequel elles
```
Comment (huats) [inline content/fr/cloud-native-apps.md:26]: gérables au lieu de manipulables

Comment (huats) [inline content/fr/cloud-native-apps.md:26]: "dans lequel elles" -> "où elles"

File: content/fr/cloud-native-tech.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+---
+
+## Ce que c'est
+
+Les technologies Cloud Natives, aussi référencées comme la pile des technologies cloud natives, sont les technologies
+utilisées pour construire des [applications cloud native](/cloud-native-apps/).
+Ces technologies permettent aux organisations de développer des applications capables de se mettre à l'échelle dans des environnements
+modernes et dynamiques tels que les clouds publics, privés et hybrides.
+Tout en tirant pleinement parti des avantages du [cloud computing](/cloud-computing/).
+Elles sont conçues dès le départ pour exploiter les capacités du cloud computings ainsi que des conteneurs, du service meshes, des micro-services. Les infrastructures immuables illustrent cette approche.
```
Comment (huats) [inline content/fr/cloud-native-tech.md:15]: enlever le s à computings

Comment (huats) [inline content/fr/cloud-native-tech.md:15]: "du service meshes" -> "des services meshes"

File: content/fr/cloud-native-tech.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+de domaines différents il touche.
+Mais à un niveau élevé, ils relèvent un ensemble de défis important :
+Les inconvénients des modèles d'exploitation informatiques traditionnels.
+Les défis incluent les difficultés à créer des applications évolutives, tolérantes aux pannes et qui sont
+capable de se réparer toute seule. Il adresse également l'utilisation inefficace de ressources entre autres.
+
+## Quel en est l'utilité
+
+Bien que chaque technologie réponde à un problème très spécifique,
+en tant que groupe, les technologies cloud natives permettent de créer des systèmes faiblement co uplés qui sont résilients, gérabels et observables..
```
Comment (huats) [inline content/fr/cloud-native-tech.md:30]: "co uplés" -> "couplés"
"gérabels" -> "gérables"

**Review by huats — CHANGES_REQUESTED** [review]

**Review by fydrah — COMMENTED** [review]
waiting for changes related to huats review

**Review by huats — APPROVED** [review]

File: content/fr/cloud-native-tech.md (hunk 3 above)
Comment (fydrah) [reply content/fr/cloud-native-tech.md:2]: @Krast76 missing that one I think, otherwise it looks good

**Review by fydrah — CHANGES_REQUESTED** [review]

File: content/fr/cloud-native-apps.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
+---
+title: Applications Cloud Native
```
Comment (fydrah) [inline content/fr/cloud-native-apps.md:2]: Missing 's' to Natives

File: content/fr/cloud-native-apps.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+status: Completed
+category: concept
+tags: ["application", "", ""]
+---
+
+## Ce que c'est
+
+Les applications Cloud Native sont des applications conçues pour tirer avantage des innovations du [cloud computing](/cloud-computing/).
```
Comment (fydrah) [inline content/fr/cloud-native-apps.md:10]: Missing 's' to Natives

**Review by fydrah — CHANGES_REQUESTED** [review]

**Review by fydrah — APPROVED** [review]
All comments resolved, LGTM

**Pull request merged**